src/components/sidebar/SafeListItem/index.tsx

```diff
@@ -54,9 +59,8 @@ const SafeListItem = ({
 
   return (
     <ListItem
-      className={css.container}
+      className={classnames(css.container, { [css.pendingButtons]: queued || signing })}
```

Comment:
`pendingButtons` is a confusing name for a container class. It's not the buttons themselves, it's the container. `withPendingActions` would be clearer.